docs/migration-guide/faq.md

```diff
@@ -0,0 +1,7 @@
+# FAQ
+
+**Is `spatial local worker launch` supported?**<br/>
+`spatial local worker launch` is not supported and will not be supported going forward. Please start and connect your worker to a deployment manually. For more information, please refer to our [API documentation](https://docs.improbable.io/reference/latest/csharpsdk/api-reference#improbable-worker-connection-class).
```

Comment:
The reference to the docs is a bit misleading, since even with `spatial local worker launch`, the worker would have had to use the same thing. The only thing spatial local worker launch did was run the command with args specified in the worker configs.

`You will need to start your worker manually as shown here [insert link to the part of the FlexibleProjectExample README where we show the commands to start a worker]` ?

Reply:
Fixed.